browser/resources/settings/brave_overrides/about_page.ts

```diff
@@ -18,39 +20,49 @@ RegisterStyleOverride(
         margin-inline-start: unset;
       }
     </style>
-  `
+  ` as HTMLTemplateElement
```

Comment:
I think if you import `html` from `polymer.min.js` you might not need this cast - we should probably update the typing on `polymer_overriding` so this works though :laughing:

Reply:
Yup! Fixed here 19a28c6c854495eb5bcacc2d5c32e2c42ec2660e